testdata/project-v3/api/v1/webhook_suite_test.go

```diff
@@ -28,7 +28,7 @@ import (
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 
-	admissionv1beta1 "k8s.io/api/admission/v1beta1"
+	admissionv1 "k8s.io/api/admission/v1"
```

Comment:
Is your goal here to begin to use "k8s.io/api/admission/v1" instead of "k8s.io/api/admission/v1beta1"?

Reply:
Could you please clarify that in the description?
We also need to check when the k8s.io/api/admission/v1 was introduced, deprecated and will no longer be served. Could you please check the k8s docs and add on the first comment?

Additionally, what happens if I have a project scaffold with the previous versions using  v1beta1 and then I upgrade the Kubebuilder for a future release with these changes and began to scaffold using v1 for the new api/webhooks create? Will I face any issue in having apis/webhooks using v1beta1 and the others ( new scaffold using v1? Should we address any recommendations for users to migrate/update their projects? What are the steps?

Reply:
As I described [here](https://github.com/kubernetes-sigs/kubebuilder/pull/2619#issuecomment-1098989462), I changed this PR is not for this change.